pkg/aws/eni/limits.go

```diff
@@ -252,7 +260,34 @@ var limits = map[string]Limits{
 }
 
 // GetLimits returns the instance limits of a particular instance type
-func GetLimits(instanceType string) (limit Limits, ok bool) {
+func GetLimits(instanceType string) (limit Limits, ok bool, err error) {
+	if len(option.Config.AwsInstanceLimitMapping) != 0 {
+		limitString, ok := option.Config.AwsInstanceLimitMapping[instanceType]
+		if ok {
+			limit, err = parseLimitString(limitString)
```

Comment:
It's not ideal to parse the string from scratch every time the limit is required. Can we hide this behind a `sync.Once`? The validation of the argument should happen at bootstrap time, otherwise the agent will fail with an invalid limit at an arbitrary time, potentially in the middle of operations.

Reply:
Thanks for pointing out! Added a function to update limits object and called it at bootstrap time, agent will exit if update limits failed.